app/v2/notifications/post_notifications.py

```diff
@@ -197,17 +200,25 @@ def post_notification(notification_type):
 
 
 def process_sms_or_email_notification(*, form, notification_type, api_key, template, service, reply_to_text=None):
-    form_send_to = form['email_address'] if notification_type == EMAIL_TYPE else form['phone_number']
-    send_to = validate_and_format_recipient(send_to=form_send_to,
-                                            key_type=api_key.key_type,
-                                            service=service,
-                                            notification_type=notification_type)
-    # Do not persist or send notification to the queue if it is a simulated recipient
-    simulated = simulated_recipient(send_to, notification_type)
+    form_send_to = form["email_address" if (notification_type == EMAIL_TYPE) else "phone_number"]
```

Comment:
is `"phone number"` supposed to be `form["phone_number"]` ?

Reply:
look a little closer. the syntax is confusing. it says use form["email_address"] if notification_type == EMAIL_TYPE else form["phone_number"]. The condition is just inside form[condition]